R/contingencytables.R

```diff
@@ -202,39 +202,42 @@ ContingencyTables <- function(jaspResults, dataset, options, ...) {
   }
 }
 
-.crossTabLogOdds <- function(jaspResults, dataset, options, analyses, ready) {
-  if(!options$oddsRatio)
-    return()
 
-  for (i in 1:nrow(analyses)){
+.crossTabOdds <- function(jaspResults, dataset, options, analyses, ready){
+  if (!options$oddsRatioNormal)
+    return ()
+  
+  for (i in 1:nrow(analyses)) {
     analysis <- analyses[i,]
-    analysisContainer <- jaspResults[[paste0("container", i)]]
-    if (!is.null(analysisContainer[["crossTabLogOdds"]]))
+    analysisContainer <- jaspResults[[paste0("container",i)]]
+    if (!is.null(analysisContainer[["crossTabOdds"]]))
       next
-
+    
     # Create table
-    crossTabLogOdds <- createJaspTable(title = gettext("Log Odds Ratio"))
-    crossTabLogOdds$dependOn(c("oddsRatio", "oddsRatioConfidenceIntervalInterval", "oddsRatioHypothesis", "rowOrder", "columnOrder"))
-    crossTabLogOdds$showSpecifiedColumnsOnly <- TRUE
-    crossTabLogOdds$position <- 3
-
-    ci.label <- gettextf("%.0f%% Confidence Intervals", 100*options$oddsRatioConfidenceIntervalInterval)
-
+    crossTabOdds <- createJaspTable(title = gettext("Odds Ratio")) 
```

Comment:
I'd also change the title of the table depending on whether the values are on a log scale or not.